### PR TITLE
fixed buff offset

### PR DESCRIPTION
**Pull request by noobLue**

### CODE REVIEW

File: GameOffsets/LifeComponentOffsets.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ namespace GameOffsets
     public struct LifeComponentOffsets
     {
         [FieldOffset(0x8)] public long Owner;
-        [FieldOffset(0x180)] public NativePtrArray Buffs;  // wrong?
+        [FieldOffset(0x158)] public NativePtrArray Buffs;
```
Comment (mm3141) [inline GameOffsets/LifeComponentOffsets.cs:10]: I don't get good data for buffs for that offset. It looks like a vector, but probably for something different?

Comment (0xE0D59) [reply GameOffsets/LifeComponentOffsets.cs:10]: Compiled and checked it, not getting buffs either.

Comment (noobLue) [reply GameOffsets/LifeComponentOffsets.cs:10]: Yeah I jumped the gun on that, I think buffs moved to a new component "Buffs"

Comment (mm3141) [reply GameOffsets/LifeComponentOffsets.cs:10]: At least there is a component named like that

Comment (mm3141) [reply GameOffsets/LifeComponentOffsets.cs:10]: can confirm, list is on Buffs component +0x158

Comment (noobLue) [reply GameOffsets/LifeComponentOffsets.cs:10]:
```
struct BuffString {
	[0x0 ]wchar_t* Name; 
	[]wchar_t* Description;
};

struct Buff {
	[0x0] BuffString* buffString;
       // still need to check if Maxtime, time and charges are still here
};


// new Buffs-component
struct Buffs {
	[0x158] std::vector<Buff*> Buffs;
};
```

Yep, looks something like this, I think.

Comment (mm3141) [reply GameOffsets/LifeComponentOffsets.cs:10]: I put up a PR https://github.com/Queuete/ExileApi/pull/120

Comment (noobLue) [reply GameOffsets/LifeComponentOffsets.cs:10]: Sweet!

**Pull request closed without merging**